Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -147,7 +147,7 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
     let nav = UINavigationController(rootViewController: vc)
     nav.modalPresentationStyle = .formSheet
 
-    self.present(nav, animated: true, completion: nil)
+    self.present(nav, animated: true) { self.viewModel.inputs.addNewCardPresented() }
```

Comment:
Doesn't cause a retain cycle because we're not holding a reference to the closure 👍